api/src/commonMain/kotlin/dev/kord/cache/api/observables/EntryCache.kt

```diff
@@ -0,0 +1,18 @@
+package dev.kord.cache.api.observables
+
+/**
+ * A cache for a single value of type [T].
+ */
+public interface EntryCache<T : Any> {
```

Comment:
what should `EntryCache` be used for and what should `Cache` be used for? is there a reason why there are two interfaces now?